package.json

```diff
@@ -123,8 +123,8 @@
     "screwdriver-scm-github": "^12.6.0",
     "screwdriver-scm-gitlab": "^3.1.0",
     "screwdriver-scm-router": "^7.1.0",
-    "screwdriver-template-validator": "^8.2.0",
-    "screwdriver-workflow-parser": "^4.4.1",
+    "screwdriver-template-validator": "file:../template-validator/screwdriver-template-validator-8.0.0.tgz",
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/template-validator/pull/42 is merged